src/Umbraco.Web.UI.Client/src/views/common/overlays/contenttypeeditor/compositions/compositions.html

```diff
@@ -15,7 +15,13 @@
 </div>
 
 <umb-empty-state
-    ng-if="model.availableCompositeContentTypes.length === 0"
+    ng-if="model.contentType.firstContentType"
+    position="center">
+    There are no content types available to use as a composition.
+</umb-empty-state>
+
+<umb-empty-state
+    ng-if="model.availableCompositeContentTypes.length === 0 && !model.contentType.firstContentType"
```

Comment:
Please note this will conflict with:
https://github.com/umbraco/Umbraco-CMS/pull/1007/files#diff-c5c287c9f99c717019173e1015c16dc7L157
when it is merged in ...